internal/hardware/validator.go

```diff
@@ -46,6 +46,10 @@ type ValidatorCfg struct {
 	MaximumAllowedTimeDiffMinutes    int64                        `envconfig:"HW_VALIDATOR_MAX_TIME_DIFF_MINUTES" default:"4"`
 	InstallationDiskSpeedThresholdMs int64                        `envconfig:"HW_INSTALLATION_DISK_SPEED_THRESHOLD_MS" default:"10"`
 	VersionedRequirements            VersionedRequirementsDecoder `envconfig:"HW_VALIDATOR_REQUIREMENTS" default:"[]"`
+	MaxNetworkLatencyMsMaster        float64                      `envconfig:"HW_VALIDATOR_MAX_NETWORK_LATENCY_MS_MASTER" default:"100"`
```

Comment:
Understood.

Reply:
I will keep them here for now. Once your [PR](https://github.com/openshift/assisted-service/pull/1458) is merged, I'll raise a new PR and move its contents to the new json structure. Otherwise, if we merge this PR without default values, it won't be possible to create a cluster due to the network latency requirements being default to 0 for all roles. WDYT?

Reply:
:+1: